modules/exploits/unix/sonicwall/sonicwall_xmlrpc_rce.rb

```diff
@@ -0,0 +1,136 @@
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name' => "SonicWall Global Management System Virtual Appliance
+                 DispatcherHandler set_time_zone Unauthenticated
+                 Remote Command Injection Vulnerability",
```

Comment:
That's quite a long module name. Suggest something like:

* SonicWall Global Management System DispatcherHandler set_time_zone Unauthenticated RCE
* SonicWall Global Management System DispatcherHandler set_time_zone Unath RCE
* SonicWall Global Management System XMLRPC set_time_zone Unath RCE
* SonicWall Global Management System XMLRPC Unauthenticated RCE
* SonicWall GMSVP DispatcherHandler set_time_zone Unauthenticated RCE

As:

* `vulnerability` is implied, by virtue of being an exploit module.
* `virtual appliance` can likely be removed without removing meaning from the name.
* `Remote Command Injection` can be shortened to `RCE` while remaining intuitive.

Also, the filename `sonicwall_xmlrpc_rce.rb` is much simpler. Generally the module name and file name should be as similar as possible.

Reply:
Keeping closer to the module name, I went with SonicWall Global Management System XMLRPC set_time_zone Unath RCE